src/cmds/Wc.my

```diff
@@ -0,0 +1,32 @@
+package embox.cmd
+
+@AutoCmd
+@Cmd(name = "wc",
+	help = "wc util(copies the functionality of wc from GNU coreutils)",
+	man = '''
+		NAME
+			wc - wc util(copies the functionality of wc from GNU coreutils)
+		SYNOPSIS
+			wc FILE_NAME
```

Comment:
It's actually `wc [OPTIONS] [FILE_NAME]`.

It's important because in this implementation you really assume that file names are going just after the options (if they will be mixed like `wc file1 -c file2` it will fail).